python-server/test/test_Camera.py

```diff
@@ -0,0 +1,141 @@
+import pytest
```

Comment:
Minor comment that people module names should be snake_case (test_camera.py instead of test_Camera.py)

Reply:
-Revised to test_camera.py